ResearchKit/Common/ORKHelpers_Internal.h

```diff
@@ -29,11 +29,18 @@
  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-
+#if TARGET_OS_IOS
 @import UIKit;
+#else
```

Comment:
Thought about it but frankly, decided against it as "premature optimization".

Once we make the break from ResearchKit/ResearchKit (or get write permission for that fork), then I want to do a reorganization of the framework for readability and maintenance. At that point, we could support `ORKStep`, `ORKResult` and  `ORKTask` on the watch and on the Apple tv, with a different defines and helpers header for each.